rust/integration-testing/src/bin/arrow-json-integration-test.rs

```diff
@@ -68,24 +69,19 @@ fn main() {
         .value_of("json")
         .expect("must provide path to json file");
     let mode = matches.value_of("mode").unwrap();
-    let verbose = matches.value_of("verbose").is_some();
+    let verbose = true; //matches.value_of("verbose").is_some();
```

Comment:
I couldn't find a way to specify this when invoking archery, so hard-coded it for now

Reply:
`archery --quiet`, RustTester should receive a `quiet: Bool` via [kwargs](https://github.com/apache/arrow/blob/master/dev/archery/archery/integration/runner.py#L335) somehwere.

Reply:
Filed https://issues.apache.org/jira/browse/ARROW-8859 to resolve this as a follow on item